src/core/model_config_utils.cc

```diff
@@ -1364,6 +1364,17 @@ ValidateModelInput(
         "shape tensors are only supported for TensorRT platform");
   }
 
+  if (
+#ifdef TRITON_ENABLE_TENSORRT
+      (platform != kTensorRTPlanPlatform) &&
+#endif  // TRITON_ENABLE_TENSORRT
+      io.allow_ragged_batch()) {
+    return Status(
+        Status::Code::INVALID_ARG,
+        "ragged-batch input tensors are only supported for custom platform"
```

Comment:
Now supported for Tensorflow as well, and soon ORT.

"ragged-batch input tensors are not support for this platform/backend"

Reply:
Why adding these checks? This type of check ought to be done by the backend really, as now Triton doesn't aware whether the backend of the model config support these features or not.

Reply:
Removing. These changes are outdated and unnecessary